docs/node-setup.md

```diff
@@ -0,0 +1,83 @@
+# Running Node using Cosmovisor
+
+As a process manager, Cosmovisor provides us a way to upgrade the likecoin binary seamless at will.
+
+To begin with, please make sure the following executables exist and added to your PATH environment variable.
+
+### Cosmovisor
+
+Please refer to https://docs.cosmos.network/master/run-node/cosmovisor.html#setup for setup procedures
+
+### Likecoin
+
+Create a genesis build of Likecoin using the following command at the project root folder
+
+```
+make build
+```
+
+Initialize the node using the following command
+
+```
+MONIKER=<moniker> CHAIN_ID=<chain_id> make init
+```
+
+### Auto Setup (systemd)
+
+To setup a node using our setup script, please navigate to the `deploy` folder and run the following command
+
+```
+make node-setup
+```
+
+or
+
+```
+LIKED_WORKDIR=<workdir> make node-setup
+```
+
+if you wish to customize the work directory (default: $HOME )
+
+This will download the latest binary of cosmovisor and liked if not already exist and copy the service file to the systemd service folder `/etc/systemd/system/`.
+
+After the script is executed, run `sudo systemd start liked` to start the node as service.
+
+**NOTE: If any sort of liked instance is running, Please terminate them before starting a new node to prevent double signing**
+
+### Manual Setup
+
+Create a folder as a home directory for cosmovisor at the likecoin home folder
+
+```
+mkdir -p $HOME/.liked/cosmovisor/genesis/bin
+```
+
+Proceed to copy your genesis binary to the folder created above
+
+```
+cp ./build/liked $HOME/.liked/cosmovisor/genesis/bin
+```
+
+Proceed to edit `liked.service` to point the directories to the correct location
+
+Then, copy the `liked.service` file to `/etc/systemd/system/liked.service`
+
+Run `sudo service start liked` to start the node
+
+### Upgrades
+
+There are two methods to upgrading the chain version using cosmovisor.
+
+#### Automatic Upgrade
+
+An automatic upgrade will be executed if an upgrade proposal is submitted and approved with the upgrade info attached. With this method, no extra action is required and cosmovisor will be able to download the suitable binary and replace the current executable in which the process will restart itself automatically when the upgrade block is reached. However, one should monitor the upgrade process to ensure the upgrade is executed successfully.
+
+#### Manual Upgrade
+
+If a manual upgrade is expected, the new binary should be placed inside
```

Comment:
and the additional version info file?

Reply:
it doesn't seems to be mandatory but I will jog down